### PR TITLE
Adding results cli to opc

### PR DESCRIPTION
**Pull request by rupalibehera**
This adds the results cli to opc
```
[rbehera@localhost opc]$ ./bin/opc results
Environment Variables:
        TKN_RESULTS_SSL_ROOTS_FILE_PATH: Path to local SSL cert to use.
        TKN_RESULTS_SSL_SERVER_NAME_OVERRIDE: SSL server name override (useful if using with a proxy such as kubectl port-forward).

Config:
    A config file may be stored in `~/.config/tkn/results.yaml` to configure the CLI client.

    Fields:
    - address: Results API Server address
    - service_account: When specified, the CLI will first fetch a bearer token
                       for the specified ServiceAccount and attach that to Result API requests.
        - namespace: ServiceAccount namespace
        - name: ServiceAccount name
    - token: Bearer token to use for API requests. Takes priority over service_account.
    - ssl: SSL connection options
        - roots_file_path: Path to a certificate to include in the cert pool. Useful for adding allowed self-signed certs.
        - server_name_override: For testing only. Sets the grpc.ssl_target_name_override value for requests.

    Example:
    
    ```
    address: results.dogfooding.tekton.dev:443
    token: abcd1234
    ssl:
        roots_file_path: path/to/file
        server_name_override: example.com
    service_account:
        namespace: default
        name: result-reader
    ```

Usage:
opc results [command]


Flags:
  -a, --addr string        Result API server address
  -t, --authtoken string   authorization bearer token to use for authenticated requests
  -h, --help               help for results

Global Flags:
      --alsologtostderr                  log to standard error as well as files
      --log_backtrace_at traceLocation   when logging hits line file:N, emit a stack trace (default :0)
      --log_dir string                   If non-empty, write log files in this directory
      --logtostderr                      log to standard error instead of files
      --stderrthreshold severity         logs at or above this threshold go to stderr (default 2)
      --update                           update golden values
  -v, --v Level                          log level for V logs
      --vmodule moduleSpec               comma-separated list of pattern=N settings for file-filtered logging

```

### CODE REVIEW

**Comment by rupalibehera** [comment]
@chmouel @vdemeester PTAL

File: main.go [hunk 1]
```diff
@@ -9,22 +9,23 @@ import (
...
 )
 
 const (
 	pacLongDesc = `Manage your Pipelines as Code installation and resources
 See https://pipelinesascode.com for more details`
-	pacShortdesc = "Manage Pipelines as Code resources"
-	tknShortDesc = `CLI to manage Openshift Pipelines resources`
-	binaryName   = `opc`
+	pacShortdesc     = "Manage Pipelines as Code resources"
+	tknShortDesc     = `CLI to manage Openshift Pipelines resources`
```
[the remark this answers is not in the document]
Comment (rupalibehera) [reply main.go]: done

**Comment by chmouel** [comment]
one last thing, add an example to readme and it's GTG for me!

**Comment by rupalibehera** [comment]
> one last thing, add an example to readme and it's GTG for me!

opc hub: search and install from Tekton Hub

 A one liner like this  ?

**Comment by chmouel** [comment]
ya

**Review by vdemeester — APPROVED** [review]

**Comment by rupalibehera** [comment]
I noticed that when I call tkn results the output is

**Comment by vdemeester** [comment]
cc @kbaig

**Comment by vdemeester** [comment]
cc @khrm

**Comment by rupalibehera** [comment]
so I think so results-cli does not pass params here like we are doing for PAC and tkn

**Pull request merged**